docs/src/tutorials/bvp_example.md

```diff
@@ -0,0 +1,76 @@
+# Boundary Value Problems
+
+This tutorial will introduce you to the functionality for solving BVPs. Other
+introductions can be found by [checking out DiffEqTutorials.jl](https://github.com/JuliaDiffEq/DiffEqTutorials.jl). This tutorial assumes you have read the [Ordinary Differential Equations tutorial](ode_example.html).
+
+In this example we will solve the ODE that satisfies the boundary condition in the form of
+
+```math
+\begin{align}
+\frac{d}{dt} &= f(t, u) \\
+g(u) &= \vec{0}
+\end{align}
+```
+
+## Example 1: Simple Pendulum
+
+The concrete example that we are solving is the simple pendulum ``\ddot{u}+\frac{g}{L}u=0`` on the time interval ``t\in[0,\frac{\pi}{2}]``. First, we need to define the ODE
+
+```julia
+using BoundaryValueDiffEq
+const g = 9.81
+L = 1.0
+tspan = (0.0,pi/2)
+function simplependulum(t,u,du)
+    θ  = u[1]
+    dθ = u[2]
+    du[1] = dθ
+    du[2] = -(g/L)*sin(θ)
+end
+```
+
+### Boundary Condition
+
+And here is where the `Boundary` comes in. We need to write a function that calculate the residual in-place from the problem solution, such that the residual is $\vec{0}$ when the boundary condition is satisfied.
+
+```julia
+function bc1(residual, u)
+    residual[1] = u[end÷2][1] + pi/2 # the solution at the middle of the time span should be -pi/2
```

Comment:
This isn't necessarily the middle with adaptive timestepping in shooting. You should change that one to use the interpolation.